packages/main/src/RPA/MSGraph.py

```diff
@@ -0,0 +1,956 @@
+
+        :param site: Site instance obtained from \`Get Sharepoint Site\`.
+        :return: List of Drives present in the SharePoint Site.
+
+        .. code-block: robotframework
+
+            *** Tasks ***
+            List SharePoint drives
+                ${drives}    List Sharepoint Site Drives    ${site}
+                FOR    ${drive}    IN    @{drives}
+                    Log    ${drive.name}
+                    Log    ${drive.description}
+                    Log    ${drive.owner.display_name}
+                    Log    ${drive.web_url}
+                    Log    ${drive.object_id}
+                END
+        """  # noqa: W605
+        self._require_authentication()
+        return site.list_document_libraries()
+
+    @keyword
+    def list_files_in_sharepoint_site_drive(
+        self,
+        site: sharepoint.Site,
+        include_folders: Optional[bool] = False,
+        drive_id: Optional[str] = None,
+    ) -> list[drive.DriveItem]:
+        # pylint: disable=anomalous-backslash-in-string
+        """List files in the SharePoint Site drive.
+
+        If the drive_id is not informed, the default Document Library will be used.
+        The drive_id can be obtained from the keyword \`List Sharepoint Site Drives\`.
+
+        The files returned are DriveItem objects and they have additional
+        properties that can be accessed with dot-notation, see
+        \`List Files In Onedrive Folder\` for details.
+
+        :param site: Site instance obtained from \`Get Sharepoint Site\`.
+        :param include_folders: Boolean indicating if should return folders as well.
+        :param drive_id: ID of the desired drive.
+        :return: List of DriveItems present in the Site Drive.
+
+        .. code-block: robotframework
+
+            *** Tasks ***
+
+            List files in SharePoint drive
+                ${files}    List Files In Sharepoint Site Drive    ${site}
+        """  # noqa: W605
+        self._require_authentication()
+        sp_drive = self._get_sharepoint_drive(site, drive_id)
+        folder = self._get_folder_instance(sp_drive)
+        items = folder.get_items()
+        if include_folders:
+            return items
+        return [item for item in items if not item.is_folder]
+
+    @keyword
+    def download_file_from_sharepoint(
+        self,
+        target_file: Union[drive.File, str],
+        site: sharepoint.Site,
+        to_path: Union[Path, str, None] = None,
+        name: Optional[str] = None,
+        drive_id: Optional[str] = None,
+    ) -> Path:
+        # pylint: disable=anomalous-backslash-in-string
+        """Downloads file from SharePoint.
+
+        The downloaded file will be saved to a local folder.
+
+        :param target_file: `DriveItem` or file path of the desired file.
+        :param site: Site instance obtained from \`Get Sharepoint Site\`.
+        :param to_path: Destination folder of the downloaded file,
```

Comment:
possible formatting problem (I think one space is enough for the continuation line)

Reply:
Yes, fixed